src/transformers/models/rt_detr/modeling_rt_detr.py

```diff
@@ -0,0 +1,1674 @@
+# coding=utf-8
+# Copyright 2023 Facebook AI Research The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch RT_DETR model."""
+import copy
+import math
+from collections import OrderedDict
+from dataclasses import dataclass
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor, nn
+
+from transformers import AutoBackbone
+
+from ...activations import ACT2CLS
+from ...modeling_utils import PreTrainedModel
+from ...utils import (
+    ModelOutput,
+    add_start_docstrings,
+    add_start_docstrings_to_model_forward,
+    is_scipy_available,
+    is_torchvision_available,
+    logging,
+    replace_return_docstrings,
+    requires_backends,
+)
+from .configuration_rt_detr import RTDetrConfig
+
+
+if is_scipy_available():
+    from scipy.optimize import linear_sum_assignment
+
+if is_torchvision_available():
+    from torchvision.ops.misc import FrozenBatchNorm2d
```

Comment:
Hmm could we also leverage a frozen batch norm without relying on torchvision? As in DETR, Deformable DETR. This might require adding ResNet-D as a separate model to the library

Reply:
Thank you for noticing this :) 
This is a left over when I was using the original's code backbone.
Just removed.